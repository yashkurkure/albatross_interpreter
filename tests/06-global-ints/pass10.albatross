var t0 int := 0;
var f0 int := 0;
var t1 int := 0;
var f1 int := 0;
var t2 int := 0;
var f2 int := 0;
var t3 int := 0;
var f3 int := 0;
var t4 int := 0;
var f4 int := 0;
var t5 int := 0;
var f5 int := 0;
var t6 int := 0;
var f6 int := 0;
var t7 int := 0;
var f7 int := 0;
var t8 int := 0;
var f8 int := 0;
var t9 int := 0;
var f9 int := 0;
f0 := !143|40+100+86|239-50*158-73-166|99-64+240&7*214|241==143|40+100+86|239-50*158-73-166|99-64+240&7*214|241;
t0 :=  240+21|159|212-196&22<(240+21|159|212-196&22|2000000);
f1 := !225==225;
t1 :=  (92-218&182&58&18*99|2000000)>92-218&182&58&18*99;
f2 := !97&123&4+130-11&165&39&58|14<>72|240|141+166*132+101-119+11&48+136|246+117&119-146&233;
t2 :=  142<>184*66|205&3-161-248|94+252*151|246|179|175-194;
f3 := !34+202*46*190*189|217&161+81==34+202*46*190*189|217&161+81;
t3 :=  253+229&228|103+60+107*238|1&123-200*253|166|23*109<>173&43-232|86+197*75|114*159|195-12|170|230;
f4 := !221|228|96&41&144&75*68+6<(221|228|96&41&144&75*68+6|2000000);
t4 :=  110*215|80&79&231*100+232|233|126|66*45&244*68|172+87|157<(110*215|80&79&231*100+232|233|126|66*45&244*68|172+87|157|2000000);
f5 := !4==4;
t5 :=  198|130|85|98*78+53&15*185|230*184*62|160|177|185*139|49==198|130|85|98*78+53&15*185|230*184*62|160|177|185*139|49;
f6 := !49-225-229+174|254-207||1;
t6 :=  185-228|247&80+218+35|223+204*75|125*141|71&144||1;
f7 := !33==33;
t7 :=  37|255|142&45|89+244-107+195==37|255|142&45|89+244-107+195;
f8 := !145+142-12|43+163&4&67&74+92&142&79*190==145+142-12|43+163&4&67&74+92&142&79*190;
t8 :=  135+148|252*92&78-217|34|100-42*174*81|204|162+30|161|70==135+148|252*92&78-217|34|100-42*174*81|204|162+30|161|70;
f9 := !115-130*69&187|38|3&224*52|195&64<>211+254&131+255|51*77&236-239*115|25-79*245*42|198|233*59;
t9 :=  13|138*219|216|48*34+128&248|55*208-40|169&161&113|147<>42*156+242&63&139|85|231*255-205|188+234|98|147|122*193;
if (f0) { printint(144); }
if (t0) { printint(8*127&132&241-145&61|31-32+128|53); }
if (f1) { printint(199*161&76-237|47-89-122-84&85|242-9-80+198*225|45|249*215-177+207|67*42*23); }
if (t1) { printint(114*3|38*197|45&169|30*188*170|30*147-27*228-34|23|158|135|2*208-90|12*253-82+111|172-94); }
if (f2) { printint(161*53&60|17|251|214-164*4|68-97+147|134&15&52-119&9); }
if (t2) { printint(157-91|0&108|240|143-118|113*208|162*219-134-173|217); }
if (f3) { printint(190|175|81+80|22+17*204*69*212|210-171|155|113*158|215&209-43+135|228-95*66); }
if (t3) { printint(3|15-228|246+168-31-184+25|218*53&231+85*69); }
if (f4) { printint(193-8&155*137&181&133|164-158*151+80-37+131|38-231|40-59); }
if (t4) { printint(113*70|242+221&11|28|230|107&53); }
if (f5) { printint(123&198*125|191-25+203*89*196|59|132*24|63+100+72&18&253|32*112|101-157); }
if (t5) { printint(172|62+172+202|65|198+51|205+37|20+164&18|113|63&34*107*125|51|215+223+170|123+196|175*116-181); }
if (f6) { printint(19|7|185*202-208|110&76+94|215-250*59-146*70+183*179-20*178+46-49+48+154+219); }
if (t6) { printint(150|51-204-215*55|61|111&53|32-19|26|246&68-191&29&240+125|12-177*240+172|143&59&254*60-65*108|145&154*157|213|37); }
if (f7) { printint(94&174+117|14|123*4|73&90&150|154&243&11-65|171-140+123|12&88*234); }
if (t7) { printint(173-163*37&73-208*169-52&9&138+191-43&28|46|178&68&99*43&10|118*178&26|100); }
if (f8) { printint(168|12-61*98+113-108*1*193|211|77|48*228*101+39&55|209+208&180+4+145|38*73&118+143|140+102&31|113&26-19); }
if (t8) { printint(253&86-206&1*254*167*141|250*100-123*144*1&69&6*46|14|108-51-114); }
if (f9) { printint(93&228|244&44|132&136-179&186-59*119); }
if (t9) { printint(166&27|255+248+14*237-24&81*190&96*178|84&12+36-110&246*83|24+117|196-204|133|44+209); }
exit(0);
