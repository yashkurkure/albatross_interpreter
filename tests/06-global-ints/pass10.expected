144127-25756315-1-21-179352047937887-9217110-26625-49100-1
